polar-core/src/numerics.rs

```diff
@@ -70,11 +70,8 @@ impl Hash for Numeric {
         H: Hasher,
     {
         match self {
-            Numeric::Integer(i) => i,
-            Numeric::Float(f) => unsafe {
-                #[allow(clippy::transmute_ptr_to_ptr)]
-                std::mem::transmute(f)
-            },
+            Numeric::Integer(i) => *i as u64,
+            Numeric::Float(f) => f.to_bits(),
```

Comment:
You might want to hash the [discriminant](https://doc.rust-lang.org/std/mem/fn.discriminant.html) first so that ints and floats dont hash to the same thing in certain circumstances? Unless that's intentional?

Reply:
That is a good idea, thanks.